src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -216,6 +227,40 @@ public void tcp_bind(UvTcpHandle handle, ref SockAddr addr, int flags)
         {
             handle.Validate();
             Check(_uv_tcp_bind(handle, ref addr, flags));
+            if (PlatformApis.IsWindows)
+            {
+                tcp_bind_windows_extras(handle);
```

Comment:
I guess my question is what will happen if someone tries to bind to multiple network interfaces (using different SockAddrs) on the same port. Will we get an error from WSAIoctl on the second call to tcp_bind?